<launch>

    <node pkg="aruco_ros" type="single" name="aruco_simple">
    <!--
        <remap from="/camera_info" to="/gscam/camera_info" />
        <remap from="/image" to="/gscam/image_raw" />
    -->
        <remap from="/camera_info" to="/stereo/left/camera_info" />
        <remap from="/image" to="/stereo/left/image_rect_color" />
        <param name="image_is_rectified" value="True"/>
        <param name="marker_size" value="0.05"/>
        <param name="marker_id" value="582"/>
        <param name="parent_name" value="stereo_gazebo_l_stereo_camera_optical_frame"/>
        <param name="child_name" value="marker_object_frame" />
    </node>

</launch>

